fastlane/Fastfile

```diff
@@ -77,18 +77,18 @@ end
   # bundle exec fastlane code_freeze
   # bundle exec fastlane code_freeze skip_confirm:true
   #####################################################################################
-  desc "Creates a new release branch from the current trunk"
-  lane :code_freeze do | options |
+  desc 'Creates a new release branch from the current trunk'
+  lane :code_freeze do |options|
     old_version = ios_codefreeze_prechecks(options)
 
-    ios_bump_version_release()
-    new_version = ios_get_app_version()
+    ios_bump_version_release
+    new_version = ios_get_app_version
     extract_release_notes_for_version(version: new_version,
-      release_notes_file_path: File.join(PROJECT_ROOT_FOLDER, 'RELEASE-NOTES.txt'),
-      extracted_notes_file_path: File.join(PROJECT_ROOT_FOLDER, 'Simplenote', 'Resources', 'release_notes.txt'))
+                                      release_notes_file_path: File.join(PROJECT_ROOT_FOLDER, 'RELEASE-NOTES.txt'),
+                                      extracted_notes_file_path: File.join(PROJECT_ROOT_FOLDER, 'Simplenote', 'Resources', 'release_notes.txt'))
```

Comment:
Personal taste: I'd go to next line after the opening parenthesis of `extract_release_notes_for_version(`, so that all the parameters would be only indented one level (2 spaces more than the number of leading spaces from line 86) instead of shifting everything to be aligned to that `version:` first parameter that is way further to the right.

Reply:
Fixed in https://github.com/Automattic/simplenote-ios/commit/a15e5d22d216e41b22583777cdb3e5ce79235de1